certify-service/pom.xml

```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>io.mosip.kernel</groupId>
             <artifactId>kernel-keymanager-service</artifactId>
-            <version>1.2.1-java21-SNAPSHOT</version>
+            <version>${kernel.core.version}</version>
```

Comment:
mapping keymanager to kernel core version is confusing here. Can we create a separate variable for keymanager?

Reply:
Done, moved this version to another variable